policyengine_core/data/dataset.py

```diff
@@ -163,8 +166,11 @@ def save(self, year: int, key: str, values: Union[np.array, pd.DataFrame]):
                 f"Invalid data format {self.data_format} for dataset {self.label}."
             )
 
+    # This type variable represents any type that satisfies the Sequence interface.
+    TimeSeriesValues = TypeVar("TimeSeriesValues", bound=Sequence)
```

Comment:
Is this a custom-named type? By the way, the array actually isn't a time series, it's more of the structure:
{
  employment_income: {
    2022: [household_1_value, household_2_value, ...]
  }
}

Reply:
I removed this typevar. I think we can just use `Sequence`. Let me know if this looks correct now.

Reply:
Thanks, LGTM!